mitodl/micromasters
# Added nationality, removed birth city and state

#### What are the relevant tickets?
Fixes #737 
Fixes #775 

#### What's this PR do?
Removes birth city and birth state or territory, and adds a nationality field. Note that we are using a list of countries as nationalities to choose from and not using the adjectival form discussed in the issue.

#### How should this be manually tested?
Run the migrations then go to the dashboard. You should be redirected to the profile with a validation error for the empty nationality field. Choose a country, then save. The validation error should go away and the nationality should show up when you view your own profile.
